packages/react/src/class-names.js

```diff
@@ -142,3 +142,7 @@ export const ClassNames: React.AbstractComponent<
   }
   return ele
 })
+
+if (process.env.NODE_ENV !== 'production') {
+  ClassNames.displayName = 'EmotionClassNames'
```

Comment:
Shouldn't this be just `ClassNames`? It would reflect the actual name and match the inferred name if only this would be inferrable here.